data/Hints/coop2.json

```diff
@@ -22,15 +22,17 @@
         "junk":       {"order":  3, "weight": 0.0, "fixed":   1, "copies": 1},
         "barren":     {"order":  4, "weight": 0.0, "fixed":   0, "copies": 2},
         "entrance":   {"order":  5, "weight": 0.0, "fixed":   0, "copies": 2},
-        "sometimes":  {"order":  6, "weight": 0.0, "fixed":  100, "copies": 3},
+        "sometimes":  {"order":  6, "weight": 0.0, "fixed": 100, "copies": 3},
         "random":     {"order":  7, "weight": 0.0, "fixed":   0, "copies": 2},
         "item":       {"order":  0, "weight": 0.0, "fixed":   0, "copies": 2},
         "song":       {"order":  0, "weight": 0.0, "fixed":   0, "copies": 2},
         "overworld":  {"order":  0, "weight": 0.0, "fixed":   0, "copies": 2},
         "dungeon":    {"order":  0, "weight": 0.0, "fixed":   0, "copies": 2},
         "woth":       {"order":  0, "weight": 0.0, "fixed":   0, "copies": 2},
         "named-item": {"order":  8, "weight": 0.0, "fixed":   1, "copies": 3},
-        "goal":       {"order":  0, "weight": 0.0, "fixed":   0, "copies": 1}
+        "goal":       {"order":  0, "weight": 0.0, "fixed":   0, "copies": 1},
+        "dual_always":{"order": 15, "weight": 0.0, "fixed":   0, "copies": 0},
```

Comment:
Gaps in hint type orders throw errors. Set to 0 or next sequential order value. Applies to other hint distros as well.